src/config/Config.ts

```diff
@@ -73,17 +72,16 @@ export default class Config implements ConfigInterface {
   }
 
   private initialize() {
-    if (!fs.existsSync(this.CONFIG_PATH)) {
-      this.initializeWithExampleConfig();
-      return;
+    this.initializeDefaultConfig();
+    if (fs.existsSync(this.CONFIG_PATH)) {
+      this.initializeWithSavedConfig();
     }
 
-    this.initializeWithSavedConfig();
+    this.initializeFromEnvironmentVariables();
   }
 
-  private initializeWithExampleConfig() {
-    this.ensureConfigDirectoryExists();
```

Comment:
Does it still work if there is no `config.json` file?

Reply:
Jep! will create the file if there is none 🎉